<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FancySearchDialog</class>
 <widget class="QDialog" name="FancySearchDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>340</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Advanced Search</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../nota.qrc">
    <normaloff>:/img/nb.png</normaloff>:/img/nb.png</iconset>
  </property>
  <widget class="QListWidget" name="tagList">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>84</y>
     <width>241</width>
     <height>249</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>6</horstretch>
     <verstretch>5</verstretch>
    </sizepolicy>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QListWidget" name="bookList">
   <property name="geometry">
    <rect>
     <x>354</x>
     <y>84</y>
     <width>239</width>
     <height>249</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>6</horstretch>
     <verstretch>5</verstretch>
    </sizepolicy>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="noneTags">
   <property name="geometry">
    <rect>
     <x>169</x>
     <y>34</y>
     <width>79</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Except</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="allBooks">
   <property name="geometry">
    <rect>
     <x>354</x>
     <y>34</y>
     <width>45</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>All</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <attribute name="buttonGroup">
    <string>bookGroup</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="moreTagsButton">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>7</y>
     <width>87</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>More Tags</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="allTags">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>34</y>
     <width>45</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>All</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <attribute name="buttonGroup">
    <string>tagGroup</string>
   </attribute>
  </widget>
  <widget class="QCheckBox" name="anyTags">
   <property name="geometry">
    <rect>
     <x>98</x>
     <y>34</y>
     <width>55</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Any</string>
   </property>
   <attribute name="buttonGroup">
    <string>tagGroup</string>
   </attribute>
  </widget>
  <widget class="QCheckBox" name="anyBooks">
   <property name="geometry">
    <rect>
     <x>424</x>
     <y>34</y>
     <width>55</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Any</string>
   </property>
   <attribute name="buttonGroup">
    <string>bookGroup</string>
   </attribute>
  </widget>
  <widget class="QCheckBox" name="noneBooks">
   <property name="geometry">
    <rect>
     <x>495</x>
     <y>34</y>
     <width>79</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Except</string>
   </property>
  </widget>
  <widget class="QPushButton" name="searchButton">
   <property name="geometry">
    <rect>
     <x>98</x>
     <y>7</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>354</x>
     <y>7</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="moreBooksButton">
   <property name="geometry">
    <rect>
     <x>495</x>
     <y>7</y>
     <width>98</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>More Books</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="connectAnd">
   <property name="geometry">
    <rect>
     <x>252</x>
     <y>198</y>
     <width>66</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>And</string>
   </property>
   <attribute name="buttonGroup">
    <string>connectGroup</string>
   </attribute>
  </widget>
  <widget class="QCheckBox" name="connectOr">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>145</y>
     <width>66</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Or</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <attribute name="buttonGroup">
    <string>connectGroup</string>
   </attribute>
  </widget>
 </widget>
 <resources>
  <include location="../nota.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="tagGroup"/>
  <buttongroup name="bookGroup">
   <property name="exclusive">
    <bool>true</bool>
   </property>
  </buttongroup>
  <buttongroup name="connectGroup"/>
 </buttongroups>
</ui>
